### PR TITLE
Improve error message when file not found

### PR DESCRIPTION
**Pull request by perolavsvendsen**
Solve #41

### CODE REVIEW

File: src/fmu/config/utilities.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def yaml_load(filename, safe=True, tool=None):
     """
 
     if not os.path.isfile(filename):
-        raise IOError("File {} cannot be read".format(filename))
+        raise FileNotFoundError(filename)
```
Comment (berland) [inline src/fmu/config/utilities.py:28]: There is probably no point in raising this exception as such, probably exactly the same thing will happen in line 30 anyways.

Comment (jsorb) [reply src/fmu/config/utilities.py:28]: Yes, I agree. I think the entire check can be removed, as yaml will raise FileNotFoundError by itself

**Review by jsorb — APPROVED** [review]
Looks good. os is no longer used and the import can be skipped.

File: src/fmu/config/utilities.py (hunk 1 above)
Comment (perolavsvendsen) [reply src/fmu/config/utilities.py]: Yes, that was obvious (doh!). A bit quick on the trigger, trying to take this small issue in addition to another fix. Have removed the whole thing now.

**Review by jcrivenaes — APPROVED** [review]
Looks OK. Rebase on commit in #46 to fix the 3.6 test

**Pull request merged**